COMPATIBILITY.md

```diff
@@ -9,8 +9,11 @@ The below matrix shows the compatibility of the [`opensearch-py`](https://pypi.o
 | --- | --- | --- |
 | 1.0.0 | 1.0.0-1.2.4 | |
 | 1.1.0 | 1.3.0-1.3.7 | |
-| 2.0.0 | 1.0.0-2.4.1 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
-| 2.0.1 | 1.0.0-2.4.1 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
+| 2.0.0 | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
+| 2.0.1 | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
+| 2.1.0 | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
+| 2.1.1 | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
+| 2.2.0 | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |
```

Comment:
We could summarize this as 2.x | 1.x and 2.x, might be more future proof?

Reply:
> We could summarize this as 2.x | 1.x and 2.x, might be more future proof?

@dblock, I think we can merge patch versions but minor versions shouldn't be merged right? Can I write something like | 2.1.x | 1.0.0-2.6.0 | client works against Opensearch Version 1.x as long as features removed in 2.0 are not used |

Reply:
Yes, I think that's better!